lasagne/layers/base.py

```diff
@@ -239,9 +239,12 @@ def create_param(self, param, shape, name=None):
 
             return theano.shared(utils.floatX(arr), name=name)
 
+        elif isinstance(param, (int, long, float, np.float16, np.float32, np.float64)):
```

Comment:
This should be unnecessary, it will be taken care of in a separate PR without having to enumerate all possible scalar types :) See #243 and https://github.com/Lasagne/Lasagne/pull/228#discussion_r30338737